src/app/gamma-sky.component.html

```diff
@@ -1,4 +1,4 @@
-<h1>
-  {{title}}
-</h1>
-<app-aladin-map>Loading app-aladin-map ...</app-aladin-map>
+<!DOCTYPE html>
```

Comment:
Doctype directive is already here:
https://github.com/gammapy/gamma-sky/blob/master/src/index.html#L1

Remove from this component?